defaults/main.yml

```diff
@@ -1,14 +1,19 @@
 ---
 
 alpaca_from_source: no
-
 alpaca_version: master
 alpaca_clone_directory: /opt/alpaca
 
 alpaca_karaf_repos:
+  - mvn:org.apache.camel.karaf/apache-camel/2.20.4/xml/features
+  - mvn:org.apache.activemq/activemq-karaf/5.15.0/xml/features
   - mvn:ca.islandora.alpaca/islandora-karaf/LATEST/xml/features
```

Comment:
Do we want to stop now and start using tagged versions for our feature?  @ajstanley got bit by that one this morning and spun up a new playbook only to receive the new version of alpaca but the old version of everything else.

Reply:
Probably...